website/src/utils/yamlDataUtils/delete.js

```diff
@@ -8,28 +8,71 @@ export const deleteData = (path, delData, yamData) => {
         }
     } else if (path.includes('caches')){
         data = deleteEnv(delData, data)
+    } else if (path.includes('data_accesses')){
+        data = deleteDataAccessData(delData, data)
+    } else if (path.includes('entities')){
+        data = deleteEntityData(delData, data)
     }
     return data
 }
 
 const deleteConnData = (connData, yamData) => {
     const data = JSON.parse(JSON.stringify(yamData))
-    const steps = ['core_lib', 'entities']
-    let target = getValueAtPath(data, steps);
+    const entitySteps = ['core_lib', 'entities']
+    let entityTarget = getValueAtPath(data, entitySteps);
     const connName = connData[0].key
     const entities = []
-    if(target.length > 0){
-        target.forEach((entity, index) => {
+    if(entityTarget.length > 0){
+        entityTarget.forEach((entity) => {
             if(entity.connection !== connName){
                 entities.push(entity)
             }
         });
         data.core_lib.entities = entities
-        target = entities.slice(0)
+        entityTarget = entities.slice(0)
+    }
+    const dataAccessSteps = ['core_lib', 'data_accesses']
+    const daTarget = getValueAtPath(data, dataAccessSteps)
+    if(daTarget.length > 0) {
+        daTarget.forEach((dataAccess, index) => {
+            if(dataAccess.connection === connName){
+                delete daTarget[index]['connection']
```

Comment:
What do you think about `for( dataAccess of daTarget)` adding introducing a `break` statement .

Reply:
we also need the index here so forEach is a better choice?

Reply:
we can do  `for( let index in daTarget)` `in` instead of `of`
this will bring the index.

Reply:
you can leave it as is. what do you think is better?